tubular/mixins.py

```diff
@@ -138,7 +138,11 @@ class WeightColumnMixin:
         class attribute, indicates whether transformer has been converted to polars/pandas agnostic narwhals framework
     """
 
-    polars_compatible = False
+    polars_compatible = True
+
+    def classname(self) -> str:
```

Comment:
Q: I'm probably missing something but why do we need this here? any class which uses this Mixin will inherit this method from BaseTransformer?

Reply:
ah yeah you're right it wasn't doing anything, actioned here https://github.com/lvgig/tubular/pull/341/commits/65af6b10e16b820f477d114432c19d3b9b35f206

Reply:
Sorry, looks like it is still there? did you push all changes?

Reply:
sorry not sure what happened there, must have been working on a random branch, removed now https://github.com/lvgig/tubular/pull/341/commits/0c6996372f1852bfb310102a9348769bcd63743e